cartography/intel/rapid7/util.py

```diff
@@ -0,0 +1,303 @@
+"""
+cartography/intel/rapid7/util
+"""
+# pylint: disable=invalid-name,broad-except
+import json
+import logging
+import re
+from array import array
+
+import pandas
```

Comment:
You'll need to add pandas as a requirement in setupp.py. But please try to accomplish this without using pandas.